index.html

```diff
@@ -150,7 +150,7 @@ <h2>Scope</h2>
           <li>Data models for credentials, presentations, and proofs</li>
           <li>Registries for the data models</li>
           <li>Algorithms for the expression and verification of proofs that use existing cryptographic primitives</li>
-          <li>Refining multilingual support in data models</li>
+          <li>Support for multilingual representations of resources not limited to text strings that can be included by reference or by value</li>
```

Comment:
I find this text, while specific, still vague and confusing. Do we mean "RDF resources" or "Web resources" or "anything at the end of a URL"? Are we expressing the references as URLs?

Feels like less is more here and what we had before seemed fine.

Reply:
Thanks for the comment. I do not have enough brain cycle to think about this week, and I can't attend the VCWG teleconference. I welcome suggestions. If there are not, I will figure them out early next week.

Reply:
I think that the text we had was sufficient, and covers what is described here.
I would recommend keeping the original text.

Reply:
> I think that the text we had was sufficient, and covers what is described here. I would recommend keeping the original text.

Okay, let me revert the change on this.